doc/nrf/ug_thingy53.rst

```diff
@@ -0,0 +1,258 @@
+#. Select the ``thingy53_nrf5340_cpuapp`` as the target board and click :guilabel:`Generate Config`, which generates the configuration file and triggers the build process.
+
+   .. figure:: images/ug_thingy53_vscode_genconfig1.png
+      :alt: nRF Connect for VS Code, Generate Configuration window
+      :width: 80%
+
+      The :guilabel:`Generate Configuration` window, with ``thingy53_nrf5340_cpuapp`` as the board.
+
+#. When the build configuration and the build are complete, an :guilabel:`Actions` window appears in the lower left corner of VS Code.
+   In this window, you can trigger the build process, program the built sample or start a debug session.
+
+   .. figure:: images/ug_thingy53_vscode_genconfig2.png
+      :alt: nRF Connect for VS Code, Actions window
+      :width: 80%
+
+      The :guilabel:`Actions` window is open in the lower left corner
+
+#. Program the sample or application:
+
+   a. Connect the Thingy:53 to the debug out port on a 10-pin external debug probe, for example nRF5340 DK, using a 10-pin JTAG cable.
+   #. Connect the external debug probe to the PC using a USB cable.
+   #. Make sure that the Thingy:91 and the external debug probe are powered on.
+   #. Click :guilabel:`Flash` in the :guilabel:`Actions` window.
+
+Building and programming on the command line
+********************************************
+
+Before you start building |NCS| projects on the command line, you must :ref:`build_environment_cli`.
+
+To build and program the source code from the command line, complete the following steps:
+
+1. Open a command line or terminal window.
+#. Go to the specific directory for the sample or application.
+
+   For example, the full directory path is :file:`ncs/nrf/applications/machine_learning` when building the source code for the :ref:`nrf_machine_learning_app` application.
+
+#. Make sure that you have the required version of the |NCS| repository by pulling the ``sdk-nrf`` repository on GitHub as described in :ref:`dm-wf-get-ncs` and :ref:`dm-wf-update-ncs` sections.
+#. Get the rest of the dependencies using west::
+
+      west update
+
+#. Build the sample or application code as follows:
+
+   .. parsed-literal::
+      :class: highlight
+
+      west build -b *build_target* -d *destination_directory_name*
+
+   For *build_target*, use:
+
+   * :file:`thingy53_nrf5340_cpuapp` or :file:`thingy53_nrf5340_cpuapp_ns` when building samples for the application core.
+   * :file:`thingy53_nrf5340_cpunet` when building for the network core.
+
+#. Program the sample or application:
+
+   a. Connect the Thingy:53 to the debug out port on a 10-pin external debug probe, for example nRF5340 DK, using a 10-pin JTAG cable.
+   #. Connect the external debug probe to the PC using a USB cable.
+   #. Make sure that the Thingy:91 and the external debug probe are powered on.
+   #. Use the following command to program the sample or application to the device::
```

Comment:
```suggestion
   #. Use the following command to program the sample or application to the device:
```

Reply:
This is correct, it formats the command below.